google-cloud-bigquery/lib/google/cloud/bigquery/convert.rb

```diff
@@ -38,6 +38,7 @@ module Bigquery
       #   | `STRING`     | `String`                             |                                                    |
       #   | `DATETIME`   | `DateTime`                           | `DATETIME` does not support time zone.             |
       #   | `DATE`       | `Date`                               |                                                    |
+      #   | `GEOGRAPHY`  | `String`                             |                                                    |
```

Comment:
Are there any of these docs tables where we want to note that the strings are in WKT format?

Reply:
Good call. I'll see where I can fit that in.

Reply:
This class is private, but I added it everywhere else.